.travis.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/env bash
```

Comment:
There is a section on Travis CI that allows one to set environment variables and hence kept secret. These variables should not be committed.

Reply:
Found the Environment variables, moving the variables.

Reply:
Done